includes/Media/Media_Source_Taxonomy.php

```diff
@@ -213,6 +213,19 @@ public function update_callback_media_source( $value, $object ) {
 	 * @return array  Tax query arg.
 	 */
 	private function get_exclude_tax_query( array $args ): array {
+		/**
+		 * Filter whether the query should be filtered.
+		 *
+		 * @since 1.16.0
+		 *
+		 * @param bool  $enabled Whether the taxonomy check should be applied.
+		 * @param array $args    Existing WP_Query args.
+		 */
+		$enabled = apply_filters( 'web_stories_enable_exclude_query', true, $args );
```

Comment:
WDYT about a filter name like `web_stories_hide_auto_generated_attachments` that explains a bit more what this actually does?

Reply:
I like it. https://github.com/google/web-stories-wp/pull/9991/commits/1379c516123f70eb5b83dee7316f8f84a27de3c9